src/python/test/test_simplex_generators.py

```diff
@@ -0,0 +1,57 @@
+""" This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which is released under MIT.
+    See file LICENSE or go to https://gudhi.inria.fr/licensing/ for full license details.
+    Author(s):       Marc Glisse
+
+    Copyright (C) 2020 Inria
+
+    Modification(s):
+      - YYYY/MM Author: Description of the modification
+"""
+
+import gudhi
+import numpy as np
+
+
+def test_flag_generators():
+    pts = np.array([[0, 0], [0, 1.01], [1, 0], [1.02, 1.03], [100, 0], [100, 3.01], [103, 0], [103.02, 3.03]])
+    r = gudhi.RipsComplex(pts, max_edge_length=4)
+    st = r.create_simplex_tree(max_dimension=50)
+    st.persistence()
+    g = st.flag_persistence_generators()
+    assert np.array_equal(g[0], [[2, 2, 0], [1, 1, 0], [3, 3, 1], [6, 6, 4], [5, 5, 4], [7, 7, 5]])
+    assert len(g[1]) == 1
+    assert np.array_equal(g[1][0], [[3, 2, 2, 1]])
+    assert np.array_equal(g[2], [0, 4])
+    assert len(g[3]) == 1
+    assert np.array_equal(g[3][0], [[7, 6]])
+
```

Comment:
Here, maybe we could test with the result of persistence pairs. I don't know if it would be clearer.

Reply:
Could you clarify what we would test with persistence_pairs? Would we check if the generator is indeed an edge of the simplex given by persistence_pairs, with the same filtration value?

Reply:
I was thinking checking, for instance, that `st.flag_persistence_generators()[0]` and `list([pair[0][0], pair[1][0], pair[1][1]] for pair in st.persistence_pairs() if len(pair[1])==2)` are equal. But that's maybe not a good idea.

Reply:
Ah, checking that the pairs that are already of the right dimension, where the generators are trivial, are the same? That makes sense, I'll see what I can do.

Reply:
Like this?

Reply:
:+1: